server_installation/topics/operating-mode/crossdc.adoc

```diff
@@ -740,6 +791,33 @@ writing keys [[B0x033E243034396234..[39]]: org.infinispan.util.concurrent.Timeou
 +
 Those exceptions are not necessarily an issue. They may happen anytime when concurrent edit of same entity is triggered on both DCs. This can be the often case in a deployment. Usually the {project_name} server is notified about the failed operation and will retry it, so from the user's point of view, there is usually not any issue.
```

Comment:
...when a concurrent edit of the same entity is triggered...
This is common in a deployment.